src/assets/style/mixins.scss

```diff
@@ -0,0 +1,138 @@
+// 
```

Comment:
The whole file should be removed in future probably as there's autoprefixer for this.